CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
```

Comment:
Since we have release please, there should be no need to manually add a changelog

Reply:
Havent we dropped it ?